models/yang/sonic/sonic-mirror-session.yang

```diff
@@ -0,0 +1,60 @@
+module sonic-mirror-session {
+	namespace "http://github.com/Azure/sonic-mirror-session";
+	prefix sms;
+
+	import ietf-inet-types {
+		prefix inet;
+	}
+
+	organization
+		"SONiC";
+
+	contact
+		"SONiC";
+
+	description
+		"SONIC MIRROR SESSION";
+
+	revision 2019-05-15 {
+		description
+			"Initial revision.";
+	}
+
+	container sonic-mirror-session {
+
+		container MIRROR_SESSION {
+
+			list MIRROR_SESSION_LIST {
+				key "name";
+
+				leaf name {
+					type string;
+				}
+
+				leaf src_ip {
+					type inet:ipv4-address;
```

Comment:
Will change to inet:ip-address.

Reply:
The sonic-mirror-session.yang is not used in this release branch. Will remove for now. It will be added with correct definitions when we implement it.

Reply:
**Resolved. Removed unsupported sonic-mirror -session.yang as of now**